app/services/dos_to_unix_lines.rb

```diff
@@ -0,0 +1,16 @@
+class DosToUnixLines
+	def initialize path_to_file, path_to_result
+		@path1 = path_to_file
+		@path2 = path_to_result
+	end
+
+	def call
+		text = File.read(@path1)
+		converted = text.gsub /\r\n?/, "\n"
+		File.open(@path2, 'w') { |file| file.write(converted) }
+	end
+
+private
+attr_reader :path_to_file, :path_to_result
```

Comment:
These need to match the names of the instance variables. In this version you have @path1 and @path2, so the attr_readers should be :path1 and :path2